flixel/util/FlxBitmapData.hx

```diff
@@ -1,10 +1,25 @@
 package flixel.util;
 
-import flixel.util.FlxDestroyUtil.IFlxDestroyable;
+#if flash
+import flash.Vector;
+#else
+import openfl.Vector;
+#end
+
+import openfl.display.BlendMode;
+import openfl.display.StageQuality;
+import openfl.geom.Matrix;
+import openfl.utils.ByteArray;
 import openfl.display.BitmapData;
+import openfl.display.IBitmapDrawable;
 import openfl.display3D.textures.Texture;
+import openfl.filters.BitmapFilter;
+import openfl.geom.ColorTransform;
+import openfl.geom.Point;
+import openfl.geom.Rectangle;
 
 // TODO: implement it...
+// TODO: hack in openfl and lime asset management system to create FlxBitmapData objects (not just BitmapData)...
```

Comment:
i've opened issue about it on openfl repo: https://github.com/openfl/openfl/issues/1270